lib/web_ui/test/ui/paragraph_builder_test.dart

```diff
@@ -39,4 +39,11 @@ Future<void> testMain() async {
 
     expect(() => builder.build(), returnsNormally);
   });
+
+  test('build and layout a paragraph with an empty addText', () {
+    final ParagraphBuilder builder = ParagraphBuilder(ParagraphStyle());
+    builder.addText('');
+    final Paragraph paragraph = builder.build();
+    paragraph.layout(const ParagraphConstraints(width: double.infinity));
+  });
```

Comment:
It's a bit odd to have a test with no expectations. Can we add one to emphasize what the test is checking?

e.g.
```dart
expect(
  () => paragraph.layout(const ParagraphConstraints(width: double.infinity)),
  returnsNormally,
);
```

Reply:
I have a separate test for a straight up empty paragraph (with no `addText` at all) in the `line_metrics_test.dart` file. However, that one is skipped on the html renderer, because the properties of an empty paragraph actually differ between renderers. See https://github.com/flutter/flutter/issues/143331

So I'd rather just do one that doesn't read any metrics. But I think doing the `returnsNormally` thing that Mouad suggested is a good idea.